src/components/PostPageDetail/index.js

```diff
@@ -569,6 +570,8 @@ const PostPageDetailIdComponent = (props) => {
     getComment();
   };
 
+  const {followUnfollow} = usePostHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `followUnfollow` function is destructured from the `usePostHook` but there's no error handling in case the function fails. Consider adding error handling to improve robustness.
```diff
- const {followUnfollow} = usePostHook();
+ const {followUnfollow, error} = usePostHook();
+ if (error) {
+   // handle error
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->